cloudrun-malware-scanner/server.js

```diff
@@ -31,8 +31,8 @@ const PORT = process.env.PORT || 8080;
 const CLAMD_HOST = '127.0.0.1';
 const CLAMD_PORT = 3310;
 
-// 10 min timeout for scanning.
-const CLAMD_TIMEOUT = 600000;
+// 20 min timeout for scanning.
+const CLAMD_TIMEOUT = 1200000;
```

Comment:
Any reason for this increase?

Reply:
I guess if we're doubling the size, it stands to reason that we should double the timeout...

Reply:
Yeah given that the size is doubled I've increased the timeout to be doubled as well